Utils/github_workflow_scripts/github_workflow_scripts_tests/run_docs_review_test.py

```diff
@@ -0,0 +1,30 @@
+from argparse import Namespace
+
+from Utils.github_workflow_scripts.run_docs_review import run_docs_review
+
+
+class TestsRunDocsReview:
+
+    def test_file_name_includes_apostrophe(self, mocker, capsys):
+        """
+            Given:
+                - A string contains a file name with an apostrophe.
+            When:
+                - Running run_docs_review function of the run_docs_review github workflow script.
+            Then:
+                - Verify the doc reviewer gets the file name with the apostrophe correctly.
+                - Verify that the output is as expected.
+        """
+        file_name = "Packs/UnRealPack/Apostrophe's_Test.yml"  # name includes an apostrophe
```

Comment:
lets add another file to this string so the split will not be trivial

Reply:
added.